### ISSUE TITLE
032

### ISSUE THREAD

**Issue by just-nana** [thendo991/CleanCity#32]
No alt for an image.

Expected: All images should have "alts" for screen readers.
Actual: This image does not have an "alt".

Main index.html file

<img width="1518" height="451" alt="Image" src="https://github.com/user-attachments/assets/b84b3ab8-d401-4e9c-91c5-db910f863548" />
